sdk/tables/azure-data-tables/azure/data/tables/_common_conversion.py

```diff
@@ -15,14 +15,17 @@ class UTC(datetime.tzinfo):
     """Time Zone info for handling UTC"""
 
     def utcoffset(self, dt):
```

Comment:
(Cannot add comments for line #11) Why do we still have six here?

Reply:
I haven't update this branch with the latest main.

Reply:
This is a good catch! `datetime.timezone.utc` was introduced since Python 3.2.